NotificationApi/NotificationApi/Controllers/NotificationController.cs

```diff
@@ -41,7 +41,7 @@ public class NotificationController : ControllerBase
         [OpenApiOperation("GetTemplateByNotificationType")]
         [ProducesResponseType(typeof(NotificationTemplateResponse), (int)HttpStatusCode.OK)]
         [ProducesResponseType(typeof(string), (int) HttpStatusCode.InternalServerError)]
-        [ProducesResponseType((int)HttpStatusCode.BadRequest)]
+        [ProducesResponseType(typeof(string), (int)HttpStatusCode.BadRequest)]
```

Comment:
The exceptionmiddleware captures BadRequest exceptions and returns a string rather than a `ValidationProblemDetails` like the other APIs